howso/engine/trainee.py

```diff
@@ -4261,3 +4179,45 @@ def list_trainees(
 
     # picks up base
     return client.get_trainees(**params)
+
+
+def get_hierarchy(self) -> Dict:
+    """
+    Output the hierarchy for a trainee.
+
+    Returns
+    -------
+    dict of {str: dict}
+        Dictionary of the currently contained hierarchy as a nested dict
+        with False for trainees that are stored independently.
+    """
+    return self.client.get_hierarchy(self.id)
+
+
+def rename_subtrainee(
+    self,
+    new_name: str = None,
```

Comment:
Does it make sense to rename a trainee to None? Seems like this should be a required parameter.

Reply:
mistake. shouldn't be none